packages/perseus/src/widgets/__tests__/numeric-input_test.jsx

```diff
@@ -529,14 +529,18 @@ describe("unionAnswerForms utility function", () => {
     it("removes duplicates", () => {
         // arrange
         const forms = [
-            {
-                status: "correct",
-                value: 1,
-            },
-            {
-                status: "correct",
-                value: 1,
-            },
+            [
```

Comment:
So if you read the comment on `unionAnswerForms` and read the logic that calls it, it's expecting a list of list.